### PR TITLE
Fix conflicts due to 95061

### PR DESCRIPTION
**Pull request by iagarwa**

### CODE REVIEW

File: clang/lib/Driver/Driver.cpp [hunk 1]
```diff
@@ -4158,18 +4158,14 @@ class OffloadingActionBuilder final {
...
-        else
-=======
         if (ToolChains.front()->getTriple().isSPIRV()) {
           if (ToolChains.front()->getTriple().getVendor() == llvm::Triple::AMD)
             GpuArchList.push_back(CudaArch::AMDGCNSPIRV);
           else
             GpuArchList.push_back(CudaArch::Generic);
-        } else {
->>>>>>> 9acb533c38be833ec1d8daa06e127a9de8f0a5ef
+        } else if (ToolChains.front()->getTriple().isSPIR()) {
```
Comment (iagarwa) [inline clang/lib/Driver/Driver.cpp:4166]: I know it was isSPIROrSPIRV for Genric.
But for spriv its already covered in else of above if case. So i just added for SPIR here.

File: clang/lib/Driver/Driver.cpp [hunk 2]
```diff
@@ -4158,18 +4158,14 @@ class OffloadingActionBuilder final {
       // supported GPUs.  sm_20 code should work correctly, if
       // suboptimally, on all newer GPUs.
       if (GpuArchList.empty()) {
-<<<<<<< HEAD
-        if (ToolChains.front()->getTriple().isSPIROrSPIRV())
-          GpuArchList.push_back(CudaArch::Generic);
-        else
-=======
         if (ToolChains.front()->getTriple().isSPIRV()) {
```
Comment (mdtoguchi) [inline clang/lib/Driver/Driver.cpp:4161]: Use `isSPIRorSPIRV()` here
```suggestion
        if (ToolChains.front()->getTriple().isSPIRorSPIRV()) {
```

File: clang/lib/Driver/Driver.cpp [hunk 3]
```diff
@@ -4158,18 +4158,14 @@ class OffloadingActionBuilder final {
...
           if (ToolChains.front()->getTriple().getVendor() == llvm::Triple::AMD)
             GpuArchList.push_back(CudaArch::AMDGCNSPIRV);
           else
             GpuArchList.push_back(CudaArch::Generic);
-        } else {
->>>>>>> 9acb533c38be833ec1d8daa06e127a9de8f0a5ef
+        } else if (ToolChains.front()->getTriple().isSPIR()) {
+          GpuArchList.push_back(CudaArch::Generic);
+	} else {
```
Comment (mdtoguchi) [inline clang/lib/Driver/Driver.cpp:4168]:
```suggestion
        } else {
```
No need for the `else if`

**Review by mdtoguchi — COMMENTED** [review]
Give these a try

**Review by jsji — APPROVED** [review]
LGTM. Thanks.

File: clang/lib/Driver/Driver.cpp [hunk 4]
```diff
@@ -4158,18 +4158,12 @@ class OffloadingActionBuilder final {
...
-=======
-        if (ToolChains.front()->getTriple().isSPIRV()) {
+        if (ToolChains.front()->getTriple().isSPIROrSPIRV()) {
           if (ToolChains.front()->getTriple().getVendor() == llvm::Triple::AMD)
             GpuArchList.push_back(CudaArch::AMDGCNSPIRV);
           else
             GpuArchList.push_back(CudaArch::Generic);
-        } else {
->>>>>>> 9acb533c38be833ec1d8daa06e127a9de8f0a5ef
+	} else {
```
Comment (mdtoguchi) [inline clang/lib/Driver/Driver.cpp:4166]:
```suggestion
	    } else {
```
indentation fix please

**Review by mdtoguchi — APPROVED** [review]

**Pull request merged**